### ISSUE TITLE
Share web files

### ISSUE THREAD

**Issue by brunohpg** [lymon66/GM8135S-IPC-modified-firmware#2]
Hi,

I have a camera GM8135S with "npc version 14.00.00.35".
A have tried to update with your firmware, but there's no flwrite. Then I get memory positions and put the firmware.
It don't boot anymore. I have others cameras and later I'll copy spi flash memory to fix.
But I have linux access and I can start httpd. But there's no web files.
Can I install web files and add extra functions to camera?

**Comment by lymon66** [comment]
Yes, there have been reports these camera-boards now seem to have a newer/different uboot, without flwrite.
Both the custom and user-web partitions are different/modified, so you would need to find a way to upload both to get the new firmware and extra functions.
